src/lib/common/string.cpp

```diff
@@ -676,3 +676,41 @@ void strReplace(char* to, int toLen, const char* from, const char* oldString, co
 
   to[toIx] = 0;
 }
+
+
+
+/* ****************************************************************************
+*
+* servicePathCheck - 
+*/
+std::string servicePathCheck(const char* servicePath)
```

Comment:
I will try to unify both them to avoid entropy in the code. Moreover, the comment just above the function signature should describe what each function does so it will be easy to distinguish both them.